documentation/src/main/asciidoc/reference/mapper-orm-mapping-reindexing.asciidoc

```diff
@@ -116,37 +116,148 @@ that whenever the list of authors changes, the result of `getMainAuthor()` may h
+====
+
+Hibernate Search will accept the mapping above and boot successfully,
+since the inverse side of the association from `Book` to `BookCategory` is no longer deemed necessary.
+
+Only _shallow_ changes to a book's category will trigger automatic reindexing:
+
+* When a book is assigned a new category (`book.setCategory( newCategory )`),
+Hibernate Search will consider it a "shallow" change, since it only affects the `Book` entity.
+Thus, Hibernate Search will reindex the book automatically.
+* When a category itself changes (`category.setName( newName )`),
+Hibernate Search will consider it a "deep" change, since it occurs beyond the boundaries of the `Book` entity.
+Thus, Hibernate Search will *not* reindex books of that category automatically.
+The index will become slightly out-of-sync,
+but this can be solved by <<mapper-orm-indexing-massindexer,reindexing>> `Book` entities,
+for example every night.
+
+=== `ReindexingOnUpdate.NO`: disabling automatic reindexing for updates of a particular property
+
+`ReindexingOnUpdate.NO` is most useful for properties that change very frequently
+and don't need to be up-to-date in the index.
+
+It essentially tells Hibernate Search that changes to that property should not trigger automatic reindexing,
+
+For example, let's consider the mapping below:
+
+.A frequently-changing property
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/reindexing/reindexonupdate/no/incorrect/Sensor.java[tags=include;!getters-setters]
+----
+<1> The sensor name and status get updated very rarely.
+<2> The sensor value gets updated every few milliseconds
+<3> When the sensor value gets updated, we also update the rolling average over the last few seconds
+(based on data not shown here).
+====
+
+Updates to the name and status, which are rarely updated, can perfectly well trigger automatic reindexing.
+But considering there are thousands of sensors,
+updates to the sensor value cannot reasonably trigger automatic reindexing:
+reindexing thousands of sensors every few milliseconds probably won't perform well.
+
+In this scenario, however, search on sensor value is not considered critical and indexes don't need to be as fresh.
+We can accept indexes to lag behind a few minutes when it comes to sensor value.
+We can consider setting up a batch process that runs every few seconds
+to reindex all sensors, either through a <<mapper-orm-indexing-massindexer,mass indexer>>
+or <<mapper-orm-indexing-manual,other means>>.
+So we would really not mind if Hibernate Search just ignored changes to sensor values...
+
+That's what `@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.NO)` is for:
+it tells Hibernate Search to ignore the impact of updates to the `rollingAverage` property.
+See the modified mapping below:
+
+.Disabling automatic reindexing for a particular property with `ReindexOnUpdate.NO`
```

Comment:
Same here. I think that it is not clear that this row is a title in the final rendering of the page.